index.html

```diff
@@ -1420,183 +1419,189 @@ <h5>
-            <p data-lang="zh-hans" class="checkme">破折号</p>
+            <p data-lang="en"><span class="leadin">Em Dashes.</span> </p>
+            <p data-lang="zh-hans"><span class="leadin">破折号</span></p>
             <p data-lang="zh-hant"><span class="leadin">破折號</span></p>
-            <p data-lang="en"><span class="uname">U+2E3A TWO-EM DASH</span> [⸺] or <span class="uname">U+2014 EM DASH</span> [—] sometimes shows a continuation of tone or sound, an abrupt change in thought, or adding new content to the context. This punctuation takes one character height and two character widths.</p>
-            <p data-lang="zh-hans" class="checkme">破折号表示语气或声音的延续、语意的转换或行文的补充。是占两个汉字空间的<span class="uname">U+2E3A TWO-EM DASH</span> [⸺]或<span class="uname">U+2014 EM DASH</span> [—]。</p>
-            <p data-lang="zh-hant">破折號表示語氣或聲音的延續、語意的轉換或行文的補充。為佔二個漢字空間的<span class="uname">U+2E3A TWO-EM DASH</span> [⸺]或<span class="uname">U+2014 EM DASH</span> [—]。</p>
+
+            <p data-lang="en">Em Dashes <span class="uname">U+2E3A TWO-EM DASH</span> [⸺] or <span class="uname">U+2014 EM DASH</span> [—] sometimes show a continuation of tone or sound, an abrupt change in thought, or adding new content to the context. The punctuation mark takes up the space of two Hanzi.</p>
+            <p data-lang="zh-hans">破折号表示语气或声音的延续、语意的转换或行文的补充。为占二个汉字空间的<span class="uname" translate="no">U+2E3A TWO-EM DASH</span> [⸺]或<span class="uname" translate="no">U+2014 EM DASH</span> [—]。</p>
+            <p data-lang="zh-hant">破折號表示語氣或聲音的延續、語意的轉換或行文的補充。為佔二個漢字空間的<span class="uname" translate="no">U+2E3A TWO-EM DASH</span> [⸺]或<span class="uname" translate="no">U+2014 EM DASH</span> [—]。</p>
           </li>
           <li>
-            <p data-lang="en"><span class="leadin">Horizontal ellipsis.</span> </p>
-            <p data-lang="zh-hans" class="checkme"><span class="leadin">删节号／省略号</span></p>
-            <p data-lang="zh-hant"><span class="leadin">刪節號／省略号</span></p>
+            <p data-lang="en"><span class="leadin">Ellipses.</span> </p>
+            <p data-lang="zh-hans"><span class="leadin">省略号／删节号</span></p>
+            <p data-lang="zh-hant"><span class="leadin">刪節號／省略號</span></p>
+
             <p data-lang="en">Ellipses are used to indicate an truncation of text, an unfinished sentence or a break in speech.
-Usually use two <span class="uname">U+2026 HORIZONTAL ELLIPSIS</span>, making up a middle aligned series of six dots and take up two character widths.</p>
-            <p data-lang="zh-hans" class="checkme">删节号又称省略号，表示节省原文、语句未完或语气的不连续。通常连续使用两个<span class="uname">U+2026 HORIZONTAL ELLIPSIS […]</span>，呈现上占两个汉字空间、六个居中省略点。</p>
-            <p data-lang="zh-hant">刪節號又作省略號，表示節略原文、語句未完或語氣的不連續。通常連續使用兩個<span class="uname">U+2026 HORIZONTAL ELLIPSIS […]</span>，呈現上佔兩個漢字空間、六個居中省略點。</p>
+Usually use <span class="uname">U+2026 HORIZONTAL ELLIPSIS</span>, making up a middle aligned series of six dots and take up the space of two Hanzi.</p>
```

Comment:
The edit here has produced a significant change to the meaning of the text. The new text says that a single character should be used, but that its font glyph is two Hanzi characters wide.  The previous text says that two characters are used, each with a single Hanzi width.

Is this change intentional? I don't think it can be correct. In most places where i have seen this in japanese, two characters are used. None of the Chinese fonts i have on my Mac produce a single character with 6 dots.

Was the intent to say, instead: "An ellipsis in Chinese consists of six dots and takes up the space of two Hanzi characters. This is normally achieved using two U+2026 HORIZONTAL ELLIPSIS characters, side-by-side." (?)

Reply:
> Was the intent to say, instead: "An ellipsis…"

Yes, I think this is more accurate. I’ll make the changes.